fitnessllm_dataplatform/stream/strava/cloud_utils.py

```
@@ -0,0 +1,33 @@
+from enum import Enum
+
+from cloudpathlib import GSPath
+
+from fitnessllm_dataplatform.stream.strava.entities.enums import StravaStreams
+
+
+def get_strava_storage_path(
+    bucket: Enum,
+    athlete_id: str,
+    strava_model: StravaStreams | None,
+    **kwargs,
```

Ruff (0.8.2)

Missing type annotation for `**kwargs`

(ANN003)